tests/integration/helpers.py

```diff
@@ -645,7 +651,9 @@ async def ls_la_in_unit(
     return [
         line.strip("\r")
         for line in ls_output
-        if len(line.strip()) > 0 and line.split()[-1] not in [".", ".."]
+        if len(line.strip()) > 0
+        and line.split()[-1] not in exclude_files
+        and line.split()[-1] not in [".", ".."]
```

Comment:
1. You can probably ditch the `[".", ".."]` exclude by not using `-a` from `ls`.
2. Do we need the long listing (`-l`) output for something? If the idea is just to have one entry per line, we can just use `ls -1` instead. EDIT: found why later in the test. One could simplify the test to not differentiate dir/files

Reply:
great suggestion. addressed in [022a4cc](https://github.com/canonical/mysql-k8s-operator/pull/496/commits/022a4cc10ae7199d92283c84913f0531bc587385)